src/services/calendar/index.js

```diff
@@ -192,13 +188,15 @@ class Service {
 		};
 
 		return request(options).then((events) => {
-			events = (events.data || []).map((event) => Object.assign(event, {
-				title: event.summary,
-				allDay: false, // TODO: find correct value
-				start: Date.parse(event.dtstart),
-				end: Date.parse(event.dtend),
-				url: '', // TODO: add x-sc-field
-			}));
+			events = (events.data || []).map((event) =>
```

Comment:
Codacy found an issue: [Assignment to function parameter 'events'.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)